src/model.rs

```diff
@@ -129,29 +123,25 @@ pub enum Action {
 
 impl Model {
     fn load_session(&mut self, session: Session) {
-        self.root = session.root;
-        self.meta = session.meta;
-        self.grammars = session.grammars;
+        self.tabs[self.current_tab].root = session.root;
```

Comment:
can you make `self.tabs[self.current_tab]` into a standalone method like `self.current_tab`? to avoid the indexing getting messed up

Reply:
Yes, but it isn't going to be able to return a reference to the actual session, just a copy of it.
So we'll still need `self.tabs[self.current_tab]` in many places.

Reply:
Probably good practice anyway to do, say,
```
let s = self.current_session();
// manipulate s
self.tabs[self.current_tab] = s.clone();
```